apis/types/host_config.go

```diff
@@ -113,6 +113,9 @@ type HostConfig struct {
 	// Mount the container's root filesystem as read only.
 	ReadonlyRootfs bool `json:"ReadonlyRootfs,omitempty"`
 
+	// Platform for running container
+	Platform string `json:"Platform,omitempty"`
```

Comment:
Thanks for your PR.
Please change swagger.yml first, and then auto generate this file. It seems that I do not see the change of swagger.yml in your PR.